examples/src/app/example.tsx

```diff
@@ -12,7 +12,8 @@ import { MIN_DESKTOP_WIDTH } from './constants';
 const deviceTypeNames = {
     [DEVICETYPE_WEBGL1]: 'WebGL 1',
     [DEVICETYPE_WEBGL2]: 'WebGL 2',
-    [DEVICETYPE_WEBGPU]: 'WebGPU'
+    [DEVICETYPE_WEBGPU]: 'WebGPU',
+    [DEVICETYPE_NULL]: 'NULL'
```

Comment:
```suggestion
    [DEVICETYPE_NULL]: 'Null'
```

Reply:
Personal preference here but I think it's best not to capitalize here. When I see `NULL`, I think C++!